sources/packages/backend/apps/load-test-gateway/test/jest-e2e.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I don`t believe that we would be setting up E2E tests for this API. Do we need the configuration?